File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DeepSeaRobe.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor;

/* JADX INFO: loaded from: classes3.dex */
public class DeepSeaRobe extends LightArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_light_deep_sea_robe_name;
        this.idDescription = R.string.armor_light_deep_sea_robe_description;
        this.idEffect = R.string.armor_light_deep_sea_robe_effect;
        this.idImage = R.drawable.deep_sea_robe;
        this.price = 5505L;
        this.maxHp = 50;
        this.intelligence = 15;
        this.immunityToStatus = 0.4d;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/DeepSeaRobe.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor

class DeepSeaRobe : LightArmor() {
    override fun configureProperties() {
        idName = R.string.armor_light_deep_sea_robe_name
        idDescription = R.string.armor_light_deep_sea_robe_description
        idEffect = R.string.armor_light_deep_sea_robe_effect
        idImage = R.drawable.deep_sea_robe
        price = 5505L
        maxHp = 50
        intelligence = 15
        immunityToStatus = 0.4
    }
}
